cpp/assetmanager.cpp

```diff
@@ -118,4 +106,14 @@ void AssetManager::check_updates() {
 #endif
 }
 
+AssetManager::tex_ptr AssetManager::get_missing_tex() {
+	// Lazily load the missing texture.
+	if (!missing_tex) {
+		missing_tex.reset(new Texture{root->join("missing.png"), false});
+	}
+	
+	// Create pseudo owning ptr with a dummy deleter so there are no double deletes.
+	return tex_ptr(missing_tex.get(), [](Texture*) { });
```

Comment:
Not really if you want to support the (in my eyes questionable) idea of displaying an error texture instead of simply treating missing textures as a hard error.